src/Transactions/TransactionDetails/TransactionDetailsContainer.js

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { withRouter } from 'react-router-dom';
+import { get } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  LoadingPane,
+} from '@folio/stripes-acq-components';
+
+import {
+  fiscalYearResource,
+  fundsResource,
+  transactionsResource,
+  transactionByUrlIdResource,
+} from '../../common/resources';
+import { FISCAL_YEARS_API } from '../../common/const';
+import TransactionDetails from './TransactionDetails';
+
+const TransactionDetailsContainer = ({
+  history,
+  match,
+  mutator,
+  resources,
+}) => {
+  const transactionId = match.params.id;
+  const transaction = get(resources, ['transactionDetails', 'records', '0']);
+  const onClose = history.goBack;
+
+  useEffect(
+    () => {
+      mutator.transactionFunds.reset();
+      mutator.transactionDetails.reset();
+      mutator.fiscalYear.reset();
+      mutator.transactionDetails.GET()
+        .then(({ fiscalYearId, fromFundId, toFundId }) => {
+          mutator.fiscalYearId.replace(fiscalYearId);
+          mutator.fiscalYear.GET();
+
+          const fundsQuery = [fromFundId, toFundId].map(fundId => `id == ${fundId}`).join(' OR ');
+          mutator.transactionFunds.GET({ params: { query: fundsQuery } });
+        })
+        .catch(() => {
+          history.goBack();
```

Comment:
back when error? mb error pane?

Reply:
I don't have strong opinion on this, just wanted to save time because there are no clear requirements on this. So saving UI from throwing user broken app looks good to me

Reply:
removed it